sqlite.go

```diff
@@ -1036,6 +1036,18 @@ func Checkpoint(sqlconn SQLConn, dbName string, mode sqliteh.Checkpoint) (numFra
 	return numFrames, numFramesCheckpointed, err
 }
 
+// DisableFunction disables the named function on the given connection.
+// numArgs must match the number of args of the function to be disabled.
```

Comment:
As written, this will also let you "create" new previously non-existing no-op functions. That's harmless, but someone who misspells the function name they wanted to disable could easily be confused as there's no error to say "that doesn't exist".

Is it practical to look up the requested function first, and use that to report an error if someone tries to "disable" a function that isn't there?  (E.g., we could query `select name, narg from pragma_function_list where name = 'fname'`). That would also have the incidental benefit that we wouldn't need to know the argument count a priori.

Not positive it's worth it, but this should be uncommon enough that it might be worth the extra step.